src/components/Chat/chat_markup.tsx

```diff
@@ -187,10 +187,6 @@ const global_replacements: TextReplacement[] = [
 export function chat_markup(body: string, extra_pattern_replacements?: Array<TextReplacement>): Array<JSX.Element> {
     let replacements = global_replacements;
 
-    if (body.length > 1024) {
```

Comment:
I believe this had a purpose, if I recall correctly some game's had exceedingly long messages (put in by bad people), basically they were so long it'd crash some browsers. I'm cool with adding another parameter to this function, but we should have the default check still here as a safe guard.

Reply:
What if we add something like a `lengthLimit` parameter which could just be set arbitrarily high for mod notes and kept at 1024 for chat?

Reply:
That sounds great :)